client/js/widgets/widget.js

```diff
@@ -193,6 +201,24 @@ export class Widget extends StateManaged {
       }
     }
 
+    if(this.activateAnimation) {
+      this.animateProperties().forEach((prop)=>{
+        if(prop != null) {
+          const rule = (typeof prop == 'object')? prop : { property: prop };
+          if(delta[rule.property] !== undefined) {
+            if(rule.className == null)
```

Comment:
Should ensure className is string.

Reply:
className now accepts multiple classes (as space separated class list and/or as array). Joining the array forces everything to string representation.